### ISSUE TITLE
Error applying Scalafix migrations in 2.13 projects

### ISSUE THREAD

**Issue by fthomas** [fthomas/scala-steward#980]
Now that ScalaCheck 1.14.1 is released I see quite a few errors applying the corresponding Scalafix migration in projects that are already using 2.13. Here is one example:
```
2019-09-19 12:54:16,229 INFO  ──────────── Nurture barambani/http4s-poc-api ────────────
2019-09-19 12:54:16,229 INFO  Clone and synchronize barambani/http4s-poc-api
2019-09-19 12:54:19,765 INFO  Find updates for barambani/http4s-poc-api
2019-09-19 12:54:54,970 INFO  Ignore org.scala-lang:scala-library:provided : 2.12.8 -> 2.12.10 -> 2.13.1 (reason: ignored globally)
2019-09-19 12:54:54,970 INFO  Ignore org.scala-lang:scala-library : 2.13.0 -> 2.13.1 (reason: ignored globally)
2019-09-19 12:54:54,971 INFO  Found 3 updates:
  com.jsuereth:sbt-pgp : 1.1.2 -> 2.0.0
  org.scalacheck:scalacheck:test : 1.14.0 -> 1.14.1
  org.scalatest:scalatest:test : 3.1.0-SNAP13 -> 3.2.0-M1
2019-09-19 12:54:54,979 INFO  Process update com.jsuereth:sbt-pgp : 1.1.2 -> 2.0.0
2019-09-19 12:54:55,289 INFO  Found PR https://github.com/barambani/http4s-poc-api/pull/116
2019-09-19 12:54:55,323 INFO  PR has commits by 'Filippo Mariotti', 'Scala Steward'
2019-09-19 12:54:55,388 INFO  Process update org.scalacheck:scalacheck:test : 1.14.0 -> 1.14.1
2019-09-19 12:54:55,565 INFO  Applying migrations: NonEmptyList(Migration(org.scalacheck,NonEmptyList(scalacheck),Version(1.14.1),NonEmptyList(github:typelevel/scalacheck/v1_14_1?sha=3fc537dde9d8fdf951503a8d8b027a568d52d055)))
2019-09-19 12:55:35,105 ERROR ──────────── Nurture barambani/http4s-poc-api ──────────── failed
java.io.IOException: 'firejail sbt -batch -no-colors ;scalafixEnable;scalafix github:typelevel/scalacheck/v1_14_1?sha=3fc537dde9d8fdf951503a8d8b027a568d52d055;test:scalafix github:typelevel/scalacheck/v1_14_1?sha=3fc537dde9d8fdf951503a8d8b027a568d52d055' exited with code 1

firejail sbt -batch -no-colors ;scalafixEnable;scalafix github:typelevel/scalacheck/v1_14_1?sha=3fc537dde9d8fdf951503a8d8b027a568d52d055;test:scalafix github:typelevel/scalacheck/v1_14_1?sha=3fc537dde9d8fdf951503a8d8b027a568d52d055 ^G[info] Loading settings for project global-plugins from scala-steward-scalafix.sbt,scala-steward.sbt ...
[info] Loading global plugins from ~/.sbt/1.0/plugins
[info] Loading settings for project http4s-poc-api-build from plugins.sbt ...
[info] Loading project definition from ~/scala-steward/workspace/repos/barambani/http4s-poc-api/project
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
[info] Loading settings for project root from build.sbt ...
[info] Set current project to http4s-poc-api (in build file:~/scala-steward/workspace/repos/barambani/http4s-poc-api/)
[info] Set current project to http4s-poc-api (in build file:~/scala-steward/workspace/repos/barambani/http4s-poc-api/)
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
[error] (Compile / scalafix) scalafix.sbt.InvalidArgument: The semanticdb-scalac compiler plugin is required to run semantic rules like v1_14_1.
[error] To fix this problem for this sbt shell session, run `scalafixEnable` and try again.
[error] To fix this problem permanently for your build, add the following settings to build.sbt:
[error]   addCompilerPlugin(scalafixSemanticdb)
[error]   scalacOptions += "-Yrangepos"
[error] Total time: 13 s, completed Sep 19, 2019 12:55:34 PM
```

I suspect the reason for this error is that Scalafix is not yet fully available for Scala 2.13. I guess at least scalafix-core for 2.13 needs to be on Maven Central. But I really have no idea.

What is also unclear to me is how we can prevent this error. Should we run Scalafix migrations only in projects that cross-build with 2.12? How would we do that? Should we skip Scalafix migrations if `scalaVersion` starts with `2.13`?

**Comment by fthomas** [comment]
I just tried this crude hack and it worked on circe/circe-derivation:
```diff
-            _ <- exec(sbtCmd(scalafixEnable :: scalafixCmds.toList), repoDir)
+            _ <- exec(sbtCmd("++2.12.10!" :: scalafixEnable :: scalafixCmds.toList), repoDir)
```
Not sure I want to commit that.

**Comment by fthomas** [comment]
I finally found the cause for the above errors and reported it upstream: https://github.com/scalacenter/scalafix/issues/997

**Comment by fthomas** [comment]
Fixed by #1122.
